torchci/rockset/metrics/__sql/tts_duration_historical.sql

```diff
@@ -0,0 +1,31 @@
+SELECT
+    FORMAT_ISO8601(
+        DATE_TRUNC(
+            :granularity,
+            job._event_time AT TIME ZONE :timezone
+        )
+    ) AS granularity_bucket,
+    AVG(DATE_DIFF(
+        'second',
+        PARSE_TIMESTAMP_ISO8601(workflow.created_at) AT TIME ZONE :timezone,
+        PARSE_TIMESTAMP_ISO8601(job.completed_at) AT TIME ZONE :timezone
+    )) as tts_avg_sec,
+    AVG(DATE_DIFF(
+        'second',
+        PARSE_TIMESTAMP_ISO8601(job.started_at) AT TIME ZONE :timezone,
+        PARSE_TIMESTAMP_ISO8601(job.completed_at) AT TIME ZONE :timezone
+    )) as duration_avg_sec,
+    CONCAT(workflow.name, ' / ', job.name) as full_name,
+FROM
+    commons.workflow_job job
+    JOIN commons.workflow_run workflow on workflow.id = job.run_id
+WHERE
+    job._event_time >= PARSE_DATETIME_ISO8601(:startTime)
+    AND job._event_time < PARSE_DATETIME_ISO8601(:stopTime)
+    AND workflow.name IN ('pull', 'trunk', 'nightly', 'periodic')
+	AND workflow.head_branch LIKE 'master'
```

Comment:
we only want to check stuff on master?

Reply:
if its all branches, there are a bunch of jobs that show up that are a result of me messing around with sharding that i think would not be a great idea to include because there are a lot of them, i could write a more complex query to filter these out, but i think that is a problem for later